internal/helper/loadbalancermachine.go

```diff
@@ -487,14 +487,19 @@ func generatePromtailConfig(
 		return "", err
 	}
 
+	proxy := ""
+	if logForward.ProxyURL != "" {
+		proxy = "\n    proxy_url: '" + logForward.ProxyURL + "'"
+	}
+
 	return `server:
   disable: true
 
 positions:
   filename: /tmp/positions.yaml
 
 clients:
-  - url: '` + logForward.LokiURL + `'
+  - url: '` + logForward.LokiURL + "'" + proxy + `
```

Comment:
I don't understand what this is supposed to do.

The proxy should be under `clients` and the key is called `proxy_url`